metapackages/auto-instrumentations-node/src/utils.ts

```diff
@@ -203,6 +205,23 @@ function getEnabledInstrumentationsFromEnv() {
   return instrumentationsFromEnv;
 }
 
+/**
+ * Returns the list of instrumentations that are disabled based on the environment variable.
+ */
+function getDisabledInstrumentationsFromEnv() {
+  if (!process.env.OTEL_NODE_DISABLED_INSTRUMENTATIONS) {
+    return [];
+  }
+
+  const instrumentationsFromEnv =
```

Comment:
nit: both vars have the same format and map the same way. Could we move it to a separate function?

Reply:
Good point, I did a small cleanup here